client/src/app/owner/owner.component.scss

```diff
@@ -12,6 +12,12 @@
   flex-grow: 1;
 }
 
+#date{
```

Comment:
Since this is on both owner and viewer pages, we could add this to note.scss instead

Reply:
Just did this, good catch.